pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -41,11 +41,13 @@
     private String brokerServiceURLTLS;
     
     // Port to use to server binary-proto request
+    // Set to -1 to disable the port.
```

Comment:
Since we have already `enableTLS`, I'd prefer to maintain consistency here and have another separate flag to disable non encrypted port.

Reply:
Also, for the same consistency reason, we should have the same behavior in broker config

Reply:
I was thinking of removing enableTLS, set default value of tls port to -1 
If user want TLS he will explicitly set the value on the tls port 

Why have two extra flags (enableTls and disableNonEncryptProtocol)?

Reply:
I'd prefer to leave it empty then, rather than -1, but have the same behavior for non-TLS port. If it's empty, than it's disabled.